core/kazoo_voicemail/src/kvm_messages.erl

```diff
@@ -86,21 +86,29 @@ count_by_owner(?MATCH_ACCOUNT_ENCODED(_)=AccountDb, OwnerId) ->
     AccountId = kz_util:format_account_id(AccountDb),
     count_by_owner(AccountId, OwnerId);
 count_by_owner(AccountId, OwnerId) ->
-    ViewOpts = [{'key', [OwnerId, <<"vmbox">>]}],
-
-    case kz_datamgr:get_results(kvm_util:get_db(AccountId), <<"attributes/owned">>, ViewOpts) of
+    ViewOptions = [{'key', [OwnerId, <<"vmbox">>]}],
+    case kz_datamgr:get_results(kvm_util:get_db(AccountId), <<"attributes/owned">>, ViewOptions) of
         {'ok', []} ->
-            lager:info("voicemail box owner is not found"),
+            lager:info("no voicemail boxes belonging to user ~s found", [OwnerId]),
             {0, 0};
-        {'ok', [Owned|_]} ->
-            VMBoxId = kz_json:get_value(<<"value">>, Owned),
-            FoldersCounts = count_per_folder(AccountId, VMBoxId),
-            normalize_count_none_deleted(VMBoxId, FoldersCounts);
+        {'ok', Boxes} ->
+            FolderQuantities = count_per_folder(AccountId),
```

Comment:
@icehess this way its one db request per MODb, instead of looping the voicemails in each MODb.  imho its more efficient this way.